pkgs/applications/editors/neovim/utils.nix

```
@@ -176,7 +176,11 @@ let
         (lib.replaceStrings [ "-" ] [ "_" ])
       ];
 
-      nvimGrammars = lib.mapAttrsToList (name: value: value.origGrammar) vimPlugins.nvim-treesitter.grammarPlugins;
+      nvimGrammars = lib.mapAttrsToList (
+        name: value:
+        value.origGrammar or builtins.throw
+          "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first"
```

```suggestion
        (value.origGrammar or builtins.throw
          "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first")
```

Not sure if this needs to be wrapped ? Seeing eval error from nixpkgs-review

---

it shouldnt need to be it works in the repl but we can do it anyway

---

I'm rerunning nixpkgs-review real quick and it didn't trigger this time...

---

Looking at it, it didn't seem like it should need to be... but, this is what I saw. 

```bash
 … while evaluating derivation 'vimplugin-lua5.1-image.nvim-1.3.0-1-unstable-2024-11-10'
...
 … while evaluating derivation 'vimplugin-treesitter-grammar-markdown_inline'
         whose name attribute is located at /home/khaneliman/.local/cache/nixpkgs-review/pr-357062/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:336:7
       … while evaluating attribute 'installQueries' of derivation 'vimplugin-treesitter-grammar-markdown_inline'
...
error: attempt to call something which is not a function but a set: { type = "derivation"; CFLAGS = «thunk»; CXXFLAGS = «thunk»; __darwinAllowLocalNetworking = false; __ignoreNulls = true; __impureHostDeps = «thunk»; __propagatedImpureHostDeps = «thunk»; __propagatedSandboxProfile = «thunk»; __sandboxProfile = «thunk»; __structuredAttrs = «thunk»; «43 attributes elided» }
       at /home/khaneliman/.local/cache/nixpkgs-review/pr-357062/nixpkgs/pkgs/applications/editors/neovim/utils.nix:179:55:
          178|
          179|       nvimGrammars = lib.mapAttrsToList (name: value: value.origGrammar or builtins.throw
             |                                                       ^
          180|           "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first"

---

lmao wut is it gone at least?

---

It looks like it thinks `or` is an argument

---

> lmao wut is it gone at least?

Yeah, I think it's fine now. Just running an official nixpkgs-review instead of a local HEAD version.